doc-content/enterprise-only/release-notes/rn-support-ref.adoc

```diff
@@ -0,0 +1,16 @@
+[id='rn-support-ref']
+= Support and compatibility
+
+This section lists supported version updates and {PRODUCT} {PRODUCT_VERSION} deprecated and removed features and components. For a comlete list of supported configurations, see
+ifdef::PAM[]
+https://access.redhat.com/articles/3405381[Red Hat Process Automation Manager 7 Supported Configurations].
+endif::[]
+ifdef::DM[]
+https://access.redhat.com/articles/3354301[Red Hat Decision Manager 7 Supported Configurations].
+endif::[]
+
+
+== Supported components
+* {JWS} 5.1.0
+* {TOMCAT} 9.0
```

Comment:
I believe that this needs changing. We support Tomcat 9 which is part of JWS 5.1.0. 
I would stay away from calling out Tomcat 9 independently of JWS 5.1.0.

Reply:
Also, why are we even mentioning this, when we reference the supported configurations page on Red Hat website? This information is available there.

Reply:
First, we had a TAM request that we create a separate Tomcat installation doc:
[BXMSDOC-4148](https://issues.jboss.org/browse/BXMSDOC-4148)
Second, I have had [BXMSDOC-2651](https://issues.jboss.org/browse/BXMSDOC-2651) on my todo list for a long time, but I do agree with you.

Reply:
Ok, I'm resurrecting the discussion in 4148. Let's keep this as is for now.